includes/environment.js

```diff
@@ -83,7 +83,7 @@ pub.pasteboard = function ( h, v ) {
 };
 
 /**
- * Returns the current page and sets it if argument page is given. Numbering starts with 1. 
+ * Returns the current page and sets it if argument page is given. Numbering starts with 1.
```

Comment:
Ok, I don't get it, apparently.
Why does the PR split view suggest for many lines (like this one), that it was updated although I did not touch it and it has seemingly the same content. What am I doing wrong?

(See screenshot)
![splitview](https://cloud.githubusercontent.com/assets/9803905/16932168/fd721d8a-4d44-11e6-94d6-4e9b6a3f5991.png)